mock-core-configs/etc/mock/templates/fedora-eln.tpl

```diff
@@ -49,7 +57,7 @@ enabled=1
 repo_gpgcheck=0
 type=rpm
 gpgcheck=1
-gpgkey=file:///usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-rawhide-primary
+gpgkey={{ rawhide_gpg_keys() }}
```

Comment:
It seems to my too codish for config. While this works and it is better as the definition is on one place only, I would prefer:
```
gpgkey=file:///usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-rawhide-primary file:///usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-{{ eln_rawhide_releasever|int }}-primary file:///usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-{{ eln_rawhide_releasever|int -1}}-primary 
```

Reply:
The code is taken from the rawhide template, and I really like to have this defined in one place.  And I hate long lines ...  but will not fight too much here :-) you tell..

I actually thought that I could cut the code out into a separate template used both by rawhide and eln templates.
But I gave up as we can not use releasever here, and I'd have to touch the rawhide template.

Reply:
> will not fight too much here

me neither :)